week7/tsmessage/src/components/CheckModal.tsx

```diff
@@ -0,0 +1,50 @@
+import styled from 'styled-components'
+
+interface SecretList{
+    hint:string; 
+    password:string
+}
+
```

Comment:
```suggestion ts
interface CheckModalProps{
    hint:string; 
    password:string
}
```
로 사용하는게 적당하지아느까?! 그냥 나의 개인적인 의견!! 왜냠 컴포넌트의 props로 사용하고 이쓰니까아!!